tests/source_storages/test_source_storages.py

```diff
@@ -107,7 +107,8 @@
 @pytest.mark.parametrize(
     "source_storage_cls", set(SOURCE_STORAGES) - {RagnaDemoSourceStorage}
 )
-def test_smoke(
+@pytest.mark.asyncio
+async def test_smoke(
```

Comment:
This is correct. We basically cheated a little previously since all of our builtin source storages were sync.